modules/apps/site/site-admin-web/src/main/resources/META-INF/resources/site/site_url.jsp

```diff
@@ -25,8 +25,20 @@ Long stagingGroupId = (Long)request.getAttribute("site.stagingGroupId");
 LayoutSet publicLayoutSet = LayoutSetLocalServiceUtil.getLayoutSet(liveGroupId, false);
 LayoutSet privateLayoutSet = LayoutSetLocalServiceUtil.getLayoutSet(liveGroupId, true);
 
-String publicVirtualHost = ParamUtil.getString(request, "publicVirtualHost", BeanParamUtil.getString(publicLayoutSet, request, "virtualHostname"));
-String privateVirtualHost = ParamUtil.getString(request, "privateVirtualHost", BeanParamUtil.getString(privateLayoutSet, request, "virtualHostname"));
+//// The list of locales available for the site
+
+// Set<Locale> availableLocales = LanguageUtil.getAvailableLocales(liveGroupId);
```

Comment:
hey @travisrcory see the comments for some details on how you'd get/manipulate the available locales.